components/mdx/Table.js

```diff
@@ -0,0 +1,7 @@
+export default function Table(props) {
+  return (
+    <div className="w-full overflow-auto">
+      <table {...props}></table>
+    </div>
+  );
+}
```

Comment:
Closing blank line is recommended by GitHub

```suggestion
}

```